code/go/0chain.net/smartcontract/storagesc/collect_reward.go

```diff
@@ -52,9 +59,29 @@ func (ssc *StorageSmartContract) collectReward(
 			"error saving user stake pool, %v", err)
 	}
 
-	if err := sp.save(ssc.ID, providerId, balances); err != nil {
+	if err := sp.save(ssc.ID, providerID, balances); err != nil {
 		return "", common.NewErrorf("pay_reward_failed",
 			"error saving stake pool, %v", err)
 	}
+	if minted == 0 {
+		return "", nil
+	}
+
+	conf, err := ssc.getConfig(balances, true)
+	if err != nil {
+		return "", common.NewErrorf("allocation_creation_failed",
+			"can't get config: %v", err)
+	}
+	conf.Minted += minted
```

Comment:
why do we store minted in conf, just wondering

Reply:
[Historic reasons.](https://github.com/0chain/0chain/commit/4fe5ad6ed3ce21c717195dfd71e45d3759382bb4#diff-f6ceb560829f2e13abe6eb715c42052a505a9ad43472949a93a4b9e2eb7a221bR53)